walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/resolvers/OpenIdURLRequestResolver.kt

```diff
@@ -39,7 +40,11 @@ internal class OpenIdURLRequestResolver(val libraryConfiguration: LibraryConfigu
         if (verifiedIdRequestInput !is VerifiedIdRequestURL) throw UnSupportedVerifiedIdRequestInputException(
             "Provided VerifiedIdRequestInput is not supported."
         )
-        return resolveOpenId4VCIRequest(verifiedIdRequestInput)
+        if (libraryConfiguration.isPreviewFeatureEnabled(PreviewFeatureFlags.FEATURE_FLAG_OPENID4VCI_ACCESS_TOKEN)
+            || libraryConfiguration.isPreviewFeatureEnabled(PreviewFeatureFlags.FEATURE_FLAG_OPENID4VCI_PRE_AUTH)
+        )
+            return resolveOpenId4VCIRequest(verifiedIdRequestInput)
+        return OpenIdResolver.getRequest(verifiedIdRequestInput.url.toString(), preferHeader)
```

Comment:
From my experience with extension work, the original OpenIDResolver defers fully to the SDK, which can't forward partial requests to extensions. Theres a seperate check in resovleOpenId4VCIRequest that will fallback to OpenIdResolver's validateRequest.

Reply:
If the preview is disabled, we would want to take Issuance v1 path and not use OpenID4VCI. If we use resolveOpenId4VCIRequest always, there will be no option to turn it off.